bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Tree.java

```diff
@@ -7928,8 +7928,8 @@ LRESULT wmNotifyHeader (NMHDR hdr, long wParam, long lParam) {
 							GC gc = createNewGC(nmcd.hdc, data);
 							Rectangle imageBounds = DPIUtil.autoScaleBounds(columns[i].image.getBounds(), this.getZoom(), 100);
 							int y = Math.max (0, (nmcd.bottom - imageBounds.height) / 2);
-							gc.drawImage (columns[i].image, DPIUtil.autoScaleDown(x), DPIUtil.autoScaleDown(y));
-							x += imageBounds.width + 12;
+							x += imageBounds.width + DPIUtil.autoScaleUp(12, getZoom());
+							gc.drawImage (columns[i].image, DPIUtil.scaleDown(x, getZoom()), DPIUtil.scaleDown(y, getZoom()));
```

Comment:
Is it inteded that the two statements have been swapped? That seems to modify the behavior.

Reply:
hmm seems like a mistake but just thinking if adding to x after using it even has any effect. Seems weird.

Reply:
Yes I tested it. Seems consitent. You are right.